frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "crypto-js": "4.0.0",
     "dequal": "1.0.0",
     "epsg": "0.5.0",
-    "formik": "2.1.4",
+    "formik": "2.2.5",
```

Comment:
2.1.4 has known issues with initialValues not working correctly:
https://github.com/formium/formik/releases/tag/v2.1.5